pkg/operator/ceph/cluster/version_test.go

```diff
@@ -37,7 +37,7 @@ func TestDiffImageSpecAndClusterRunningVersion(t *testing.T) {
 	fakeRunningVersions := []byte(`
 	{
 		"mon": {
-			"ceph version 19.2.0 (3a54b2b6d167d4a2a19e003a705696d4fe619afc) squid (stable)": 1,
+			"ceph version 19.2.1 (3a54b2b6d167d4a2a19e003a705696d4fe619afc) squid (stable)": 1,
```

Comment:
Let's revert this file. No need to update the unit tests for every ceph version patch release.